eventserver/auth/authorizer.go

```diff
@@ -1,6 +1,6 @@
 package auth
 
 type Authorizer interface {
-	Spaces() ([]string, error)
 	Admin() (bool, error)
+	Organisations([]string) (bool, error)
```

Comment:
I would rename this interface method because the actual meaning that we are giving to it is: "Has this user permissions to get billing information for this set of orgs?"

For instance you could call it `BillingAllowedForOrgs(orgs)`

Reply:
I would rather change the name, because for me it lacks of meaning as it is. As a developer if I have to look at whatever function calls this, I would have to look at the implementation to know what is doing.

It should say something like:

 - `HasBillingAccess(orgs)`
 - `IsBillingManager(orgs)`
 - `BillingAllowed(orgs)`
 - `BillingManager(orgs)`